src/main/java/org/violetmoon/quark/addons/oddities/block/MovingMagnetizedBlock.java

```diff
@@ -102,11 +102,13 @@ public VoxelShape getCollisionShape(@NotNull BlockState state, @NotNull BlockGet
 		return tile != null ? tile.getCollisionShape(worldIn, pos) : Shapes.empty();
 	}
 
+	//Todo: Hey Ithundxr, is this even needed anymore? It seems kinda redundant and is unused. At the very least, its out of date.
```

Comment:
Yep can be removed if it's not being used, make sure to do a quick check first though